### PR TITLE
Add firefox/privacy-hub.lang

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/7768

### CODE REVIEW

**Comment by alexgibson** [comment]
Langchecker PR: https://github.com/mozilla-l10n/langchecker/pull/923

File: en-US/firefox/privacy-hub.lang [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+;Firefox Privacy Promise
+Firefox Privacy Promise
+
+
+# HTML page description
+;Firefox takes less data, keeps it safe, and with no secrets.
+Firefox takes less data, keeps it safe, and with no secrets.
+
+
+;Firefox products are designed to protect your <strong>privacy</strong>
```
Comment (peiying2) [inline en-US/firefox/privacy-hub.lang]: Is the <strong> used for the entire sentence or just one word? The demo is for the whole sentence.

File: en-US/firefox/privacy-hub.lang [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+
+# These strings are used in /firefox/privacy/
+# HTML page title
+;Firefox Privacy Promise
+Firefox Privacy Promise
+
+
+# HTML page description
+;Firefox takes less data, keeps it safe, and with no secrets.
```
Comment (peiying2) [inline en-US/firefox/privacy-hub.lang]: Seems like missing a verb here `... with no secrets.` (after takes... keeps...

File: en-US/firefox/privacy-hub.lang (hunk 1 above)
Comment (alexgibson) [reply en-US/firefox/privacy-hub.lang]: The `<strong>` tag is a placeholder for a CSS styling hook.

File: en-US/firefox/privacy-hub.lang (hunk 2 above)
Comment (alexgibson) [reply en-US/firefox/privacy-hub.lang]: This is the string provided by the copywriter - let me check back on this.

Comment (alexgibson) [reply en-US/firefox/privacy-hub.lang]: @peiying2 I’ll add a comment here with an alternate string for localisers

**Comment by alexgibson** [comment]
@peiying2 updated

**Comment by alexgibson** [comment]
Extracted locales, merging.

**Comment by alexgibson** [comment]
Pushed to production in https://github.com/mozilla-l10n/bedrock-l10n/commit/32866dc910167f1c8794ce689fac2e4c9406616f

**Pull request merged**